staramr/detection/AMRDetection.py

```diff
@@ -118,3 +126,12 @@ def get_summary_results(self):
         :return: A pd.DataFrame for a summary table of the results.
         """
         return self._summary_dataframe
+
+    def get_detailed_summary_results(self):
```

Comment:
For any new methods you add, could you remember to include Python type hints (e.g,. define the types of all input parameters and the type of the return value)?

Reply:
Yup will do, added linting in commit 1110972

Reply:
Could you add the Python return type here (should be `DataFrame`).